packages/@react-spectrum/dropzone/stories/DropZone.stories.tsx

```diff
@@ -29,6 +29,13 @@ import {SpectrumDropZoneProps} from '../src/DropZone';
 import styles from './styles.css';
 import Upload from '@spectrum-icons/illustrations/Upload';
 
+interface FileFilledSource {
+  src?: string;
+  id?: number;
+  type: string;
+  name: string;
+}
+
```

Comment:
I'm not happy that I had to declare this interface for a Storybook file.

Reply:
not a problem, also expected and informs us what people will need to do in their own projects in order to use our stuff